Description: smbus2 is a pure Python implementation of the python-smbus package
Forwarded: not-needed
Author: pi-top <deb-maintainers@pi-top.com>
Last-Update: 2021-09-05

--- a/pitopd/pthub/pthub_i2c.py
+++ b/pitopd/pthub/pthub_i2c.py
@@ -4,7 +4,7 @@
 from time import sleep
 
 from pitop.common.counter import Counter
-from smbus2 import SMBus
+from smbus import SMBus
 
 logger = logging.getLogger(__name__)
 
--- a/pitopd/ptpulse/ptpulse.py
+++ b/pitopd/ptpulse/ptpulse.py
@@ -2,7 +2,7 @@
 from math import pow
 
 from pitop.common.common_ids import DeviceID
-from smbus2 import SMBus
+from smbus import SMBus
 
 from ..sys_config import HDMI, I2S, UART
 
--- a/pitopd/ptspeaker/ptspeaker.py
+++ b/pitopd/ptspeaker/ptspeaker.py
@@ -5,7 +5,7 @@
 
 from pitop.common.common_ids import DeviceID
 from pitop.common.current_session_info import get_user_using_first_display
-from smbus2 import SMBus
+from smbus import SMBus
 
 from .sys_config import HDMI, I2C, I2S
 
